integrations/doubleclick-floodlight/lib/index.js

```diff
@@ -105,7 +106,11 @@ Floodlight.prototype.track = function(track) {
 
         if (Array.isArray(segmentProp)) {
           segmentProp = segmentProp.pop();
-          segmentPropValue = find(track.json(), segmentProp);
+          if (segmentProp === 'userId') {
```

Comment:
Is there any reason we shouldn't support other top-level values via `track.proxy()`?

Reply:
They are supported by this: `segmentPropValue = find(track.json(), segmentProp);` which is the existing behavior. It is just that `userId` is stored on the analytics object not the track.

Reply:
Ahh, okay. Got it.

Reply:
Updated the description of the change change to make it a little more explicit/clear. 👍